.github/workflows/ci-tlm.yml

```diff
@@ -0,0 +1,75 @@
+name: CI TLM
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  test:
+    name: "Test: Python ${{ matrix.python }} on ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
+    if: github.event.comment.body == '/test-tlm'  # only run the test if /test-tlm is commented on the PR
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        python:
+          - "3.8"
+          - "3.9"
+          - "3.10"
```

Comment:
Is there a reason why we're only testing on these 3 versions of python here, and then python 3.11 for the property tests below?

Reply:
Legacy, the tests were never ran on python 3.11. I can extend the prior normal tests to python 3.11 as well we would increase the number of calls from 180 -> 240. I did not initially do that to avoid the call increase and since the property tests are testing the usage already.

Property tests should only run on one python version since it is ~200-400 calls to the TLM. If we did this for all versions/all OS it would be too many calls

Reply:
Yeah I added the other versions. Mostly to do with throwing errors/concurency timing but its not too many extra calls <200 for a more thorough test

Reply:
Are these mostly running on `gpt-4o-mini`? If so then yeah ~200 calls is not an issue at all

Reply:
Yep!